tests/integration/test_sqs.py

```diff
@@ -1151,6 +1151,32 @@ def test_dead_letter_queue_config(self, sqs_client, sqs_create_queue):
 
         assert queue_url
 
+    def test_dead_letter_queue_list_sources(self, sqs_client, sqs_create_queue):
+        queue_name = f"queue-{short_uid()}"
+        queue_name_2 = f"queue-{short_uid()}"
+        dead_letter_queue_name = f"dead_letter_queue-{short_uid()}"
```

Comment:
nit: we don't need to generate names manually, `sqs_create_queue` will automatically generate one.

Reply:
:+1: